migrations/frontend/1684771948_add_domain_to_github_app/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE github_apps
+    ADD COLUMN IF NOT EXISTS domain TEXT NOT NULL DEFAULT 'repo';
```

Comment:
Can we change it to `repos`, so that it matches the `batches` plural format that you want to do? :)
```suggestion
    ADD COLUMN IF NOT EXISTS domain TEXT NOT NULL DEFAULT 'repos';
```